tests/interfaces/test_autograd.py

```diff
@@ -893,7 +893,8 @@ def circuit(x, y, z):
     def test_non_diff_wires_argument(self, w, expected_res, expected_grad, tol):
         """Test that passing wires as a non-differentiable positional
         argument works correctly."""
-        dev = qml.device("default.qubit", wires=2)
+        dev = qml.device("default.qubit", wires=[qml.numpy.array(0, requires_grad=False),
+                                                 qml.numpy.array(1, requires_grad=False)])
```

Comment:
This test relies on the fact that in autograd we currently convert positional arguments to `pennylane.numpy` arrays to be backwards compatible.

Reply:
Nice! And this test verifies that #1030 is solved, from what I can tell?

Reply:
Yes, I tried and it works. @glassnotes is also happy with this solution.